release-doc-in-process/all_vf.TTL

```diff
@@ -250,25 +255,32 @@ vf:inScopeOf  a             owl:ObjectProperty ;
         rdfs:comment        "Grouping around something to create a boundary or context, used for documenting, accounting, planning." .
 
 vf:under  a                 owl:ObjectProperty ;
-        rdfs:domain         [ owl:unionOf (vf:EconomicEvent vf:Commitment vf:Intent vf:Fulfillment ) ] ; 
+        rdfs:domain         [ owl:unionOf (vf:EconomicEvent vf:Commitment vf:Intent vf:Fulfillment vf:Claim) ] ; 
         rdfs:label          "under" ;
         rdfs:range          vf:Agreement ;
         vs:term_status      "unstable" ;
         rdfs:comment        "Reference to an agreement between agents which specifies the rules or policies or calculations which govern this flow." .
 
-#vf:createdBy  a             owl:ObjectProperty ;
-#        rdfs:label          "created by" ;
-#        rdfs:domain         vf:Reciprocity ;
-#        rdfs:range          vf:EconomicEvent ;
-#        vs:term_status      "unstable" ;
-#        rdfs:comment        "References the economic event that fully or partially created the commitment, often based on a prior agreement." .
+vf:triggeredBy  a           owl:ObjectProperty ;
+        rdfs:label          "triggered by" ;
+        rdfs:domain         vf:Claim ;
+        rdfs:range          vf:EconomicEvent ;
+        vs:term_status      "unstable" ;
+        rdfs:comment        "References an economic event that implied the claim, often based on a prior agreement." .
 
-#vf:creates  a               owl:ObjectProperty ;
-#        rdfs:label          "creates" ;
-#        rdfs:domain         vf:Reciprocity ;
-#        rdfs:range          vf:Commitment ;
-#        vs:term_status      "unstable" ;
-#        rdfs:comment        "References the commitment that was fully or partially created because of the economic event, often based on a prior agreement." .
+vf:settles  a               owl:ObjectProperty ;
+        rdfs:label          "settles" ;
+        rdfs:domain         vf:Settlement ;
+        rdfs:range          vf:Claim ;
+        vs:term_status      "unstable" ;
+        rdfs:comment        "References a claim that fully or partially settled by the economic event.
```

Comment:
Missing closing `"` which causes travis build to fail. I can setup pre-commit hook to use https://www.npmjs.com/package/turtle-validator to see problems with turtle file right away.

Reply:
thanks, fixed... you don't have to set up a hook, I can get in the habit of checking the travis build